api/src/test/java/uk/gov/hmcts/bar/api/componenttests/PaymentInstructionUserFilterTest.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.bar.api.componenttests;
+
+import org.junit.Assert;
+import org.junit.Test;
+import uk.gov.hmcts.bar.api.componenttests.utils.DbTestUtil;
+import uk.gov.hmcts.bar.api.data.model.CardPaymentInstruction;
+import uk.gov.hmcts.bar.api.data.model.PaymentInstruction;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+public class PaymentInstructionUserFilterTest extends ComponentTestBase {
+
+
+    @Test
+    public void getAllPaymentInstructions() throws Exception {
```

Comment:
feels like API method, It can be named like a test scenario. It's not very obiouse what are we testing here